windows/src/engine/kmtip/README.md

```diff
@@ -0,0 +1,102 @@
+
+## Overview
+
+`kmtip` is an implementation of a Text Service for the Text Services Framework
+(TSF). This implementation is named Keyman TIP. Architecturally it sits between
+the TSF Manager and Keyman.exe.
```

Comment:
I know you explain the TIP acronym below, but it might be good to do it when it first appears.

```suggestion
(TSF). This implementation is named Keyman Text Input Processor (TIP). Architecturally
it sits between the TSF Manager and Keyman.exe.
```